build.gradle

```diff
@@ -2,16 +2,17 @@ ext {
   kotlinVersion = '1.0.5'
 
   compileSdkVersion = 25
-  buildToolsVersion = '25.0.1'
+  buildToolsVersion = '25.0.2'
 
   dep = [
-      androidPlugin: 'com.android.tools.build:gradle:2.3.0-beta1',
+      androidPlugin: 'com.android.tools.build:gradle:2.3.0-beta2',
       supportAnnotations: 'com.android.support:support-annotations:25.1.0',
       compiletesting: 'com.google.testing.compile:compile-testing:0.10',
       javaPoet: 'com.squareup:javapoet:1.8.0',
       moshi: 'com.squareup.moshi:moshi:1.3.1',
       rxjava: 'io.reactivex:rxjava:1.2.3',
       jsr305: 'com.google.code.findbugs:jsr305:3.0.1',
+      jsr205: 'javax.annotation:jsr250-api:1.0',
```

Comment:
for the `@Generated` annotation. For some reason that couldn't be find anywhere else.
Do you know if that's in any of the other deps already?